src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/AldersvilkårVurdering.kt

```diff
@@ -0,0 +1,93 @@
+package no.nav.tilleggsstonader.sak.vilkår.vilkårperiode
+
+import no.nav.tilleggsstonader.sak.infrastruktur.exception.feilHvis
+import no.nav.tilleggsstonader.sak.opplysninger.grunnlag.Grunnlagsdata
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.MålgruppeType
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.faktavurderinger.SvarJaNei
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.dto.LagreVilkårperiode
+import java.time.LocalDate
+
+object AldersvilkårVurdering {
+    fun vurderAldersvilkår(
+        vilkårperiode: LagreVilkårperiode,
+        grunnlagsData: Grunnlagsdata,
+    ): SvarJaNei {
+        val fødselsdato = grunnlagsData.grunnlag.fødsel?.fødselsdato
+
+        feilHvis(fødselsdato == null) { "Kan ikke vurdere aldersvilkår uten å vite fødselsdato til bruker" }
+
+        val gyldig: SvarJaNei =
+            when (vilkårperiode.type as MålgruppeType) {
+                MålgruppeType.AAP, MålgruppeType.NEDSATT_ARBEIDSEVNE, MålgruppeType.UFØRETRYGD ->
+                    vurderAldersvilkårForNedsattArbeidsevne(fødselsdato, vilkårperiode)
+                MålgruppeType.OMSTILLINGSSTØNAD -> vurderAldersvilkårForOmstillingsstønad(fødselsdato, vilkårperiode)
+                MålgruppeType.OVERGANGSSTØNAD -> SvarJaNei.JA_IMPLISITT
+                MålgruppeType.DAGPENGER -> SvarJaNei.NEI
+                MålgruppeType.SYKEPENGER_100_PROSENT -> SvarJaNei.NEI
+                MålgruppeType.INGEN_MÅLGRUPPE -> SvarJaNei.NEI
```

Comment:
Burde vi kaste feil for disse? Disse skal vel ikke inneholde aldersvilkår foreløpig?

Reply:
Endret til å kaste feil istedenfor å gi "automatisk NEI": c2a32cd297b9e47880afa550abffead77e90f3ff